docs/generated/oc_by_example_content.adoc

```diff
@@ -776,6 +776,28 @@ Create a new secret based on a key file or on files within a directory
 ====
 
 
+== oc secrets new-sourcecfg
+Create a new source secret
```

Comment:
yup, unless you have your own server verification, I don't think that git will use a cacert to verify a ssh server